content/participate/report-issue/redirect.html.haml

```diff
@@ -80,13 +80,20 @@ uneditable: true
                 Report a security issue privately.
                 %a{:href => '/security/reporting/' }
                     Learn more...
-            %a.app-button.app-button--security.bl{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10180&issuetype=10103&components=17329&summary=[INSERT_PLUGIN_ID_HERE]%20Insert%20Summary%20Here' }
+            %a.app-button.app-button--security.bl#link-security{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10180&issuetype=10103&components=17329&summary=[INSERT_PLUGIN_ID_HERE]%20Insert%20Summary%20Here' }
                 Continue
 
 :javascript
     // TODO Also pass the plugin name and show it on the UI, but for now this is simpler and probably safer.
```

Comment:
TODO resolved by this

Reply:
No, not shown on UI (e.g., "Report an issue for Foo Plugin") and it's the ID, not the (display) name.